modules/perception/camera/lib/motion/plane_motion.cc

```diff
@@ -56,13 +56,10 @@ void PlaneMotion::generate_motion_matrix(base::VehicleStatus *vehicledata) {
         sqrt(vehicledata->velocity_x * vehicledata->velocity_x +
              vehicledata->velocity_y * vehicledata->velocity_y));
     float displacement = time_d * velocity;
-    trans(0) = static_cast<float>(displacement * cos(theta));
-    trans(1) = static_cast<float>(displacement * sin(theta));
-    // trans(0) = time_d * vehicledata->velocity_x;
-    // trans(1) = time_d * vehicledata->velocity_y;
+    trans << displacement, 0;
 
     motion_2d.block(0, 0, 2, 2) = rot2d.toRotationMatrix().transpose();
-    motion_2d.block(0, 2, 2, 1) = -rot2d.toRotationMatrix().transpose() * trans;
+    motion_2d.block(0, 2, 2, 1) = -trans;
```

Comment:
same here, seems keeping the original code is more generic.

Reply:
But, they are not equivalent here. More importantly, it is the `motion_3d`, not the `motion_2d` will be used later. That's the main purpose of this PR. Please double-check around here.

Reply:
same as above.